### PR TITLE
Glusterfs installation

### PR DESCRIPTION
**Pull request by erikh**
This implements glusterfs installation, I chose to do this early to break the changes into smaller parts.

This is just the ansible and Vagrant changes to install glusterfs. There is no code to drive it yet.

### CODE REVIEW

File: ansible/roles/gluster/tasks/install_gluster.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  yum: name=centos-release-gluster state=present
+
+- name: install glusterfs
+  yum: name=glusterfs-server state=present
+
+- name: configure gluster brick device
+  filesystem:
+    dev: "{{ gluster_device }}"
+    fstype: xfs
+    opts: "-i size=512"
```
Comment (dseevr) [inline ansible/roles/gluster/tasks/install_gluster.yml]: Why 512 here?  Just curious.

Comment (erikh) [inline ansible/roles/gluster/tasks/install_gluster.yml]: recommended in the quickstart guide.

Comment (erikh) [inline ansible/roles/gluster/tasks/install_gluster.yml]: http://gluster.readthedocs.io/en/latest/Quick-Start-Guide/Quickstart/

Comment (dseevr) [inline ansible/roles/gluster/tasks/install_gluster.yml]: Weird. 4096 is the default for XFS, and 512 is the minimum.  Thanks for the info.

Comment (erikh) [inline ansible/roles/gluster/tasks/install_gluster.yml]: either way we can always make this variable later. Honestly, how to deploy it for production will be @vvb and @mapuri's responsibility, not as much ours.

**Comment by dseevr** [comment]
Sorry, forgot to LGTM this

LGTM

**Pull request merged**